virt/virt-2-4-release-notes.adoc

```diff
@@ -44,6 +44,9 @@ The SVVP Certification applies to:
 //CNV-5438 - svirt-support-for-cnv
 * This release features significant security enhancements. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCCs to infrastructure containers in the cluster.
 
+//Bug 1853331 Guest Subscriptions
+* In this release, you can enable access to your Red Hat Enterprise Linux entitlement for RHEL virtual machines. Configure the `virt-who` daemon to report your running VMs in an {product-title} cluster. Then, the  Red Hat Subscription Manager in the RHEL VM can access your entitlements. Learn more about the https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/introduction_to_red_hat_subscription_management_workflows/index[Red Hat Subscription Manager].
```

Comment:
- I assume you'll also be able to do this in future releases, so: s/In this release, you can/You can now
- For external links, you should use the `link:` syntax: `link:http://othersite.com/otherpath[friendly reference text]` (mentioned in the OCP contribution guide)
- Nit: s/report your running VMs in an/report the running VMs in your
- Clarity suggestion: instead of "Then, the..." reword this sentence to say "This gives the Red Hat Subscription Manager in the RHEL VM access to your entitlements."
- Suggestion: Consider condensing this by adding the link to the RHSM docs in the first instance of "Red Hat Subscription Manager" and removing the last sentence.

Reply:
Content updated.